cli/index.js

```diff
@@ -6,31 +6,82 @@ const { hideBin } = require('yargs/helpers');
 const { mochify } = require('@mochify/mochify');
 
 const opts = yargs(hideBin(process.argv))
+  .usage(
+    '$0 [options] <spec...>',
+    'Run Mocha tests using Mochify.',
+    (local) => {
+      local
+        .example(
+          '$0 --driver puppeteer --bundle browserify "./src/**/*.test.js" ',
+          'Bundle all files matching the given spec using browserify and run them using @mochify/driver-puppeteer.'
+        )
+        .example(
+          '$0 --config package.json "./src/**/*.test.js" ',
+          'Run all tests matching the given spec using the configuration from package.json\'s "mochify" key.'
```

Comment:
The `mochify` key in `package.json` is picked up automatically. Specifying it like this would be superfluous.

Reply:
Oh that's interesting to know.